lib/hasher/src/ADLER32Strategy.cpp

```diff
@@ -0,0 +1,79 @@
+#include "ADLER32Strategy.hpp"
+#include "checksum.hpp"
+
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <iomanip>
+#include <string.h>
```

Comment:
given that this is a cpp file, this should be `<cstring>`.

Reply:
Done for all algorithms.